modules/govuk/manifests/node/s_base.pp

```diff
@@ -105,6 +105,31 @@
       fields  => {'application' => 'rkhunter'};
   }
 
+  @filebeat::prospector {
+    'apt-history':
+      paths  => '/var/log/apt/history.log',
+      tags   => ['history'],
+      fields => {'application' => 'apt'};
+    'apt-term':
+      paths  => '/var/log/apt/term.log',
+      tags   => ['term'],
+      fields => {'application' => 'apt'};
+    'dpkg':
+      paths  => '/var/log/dpkg.log',
+      fields => {'application' => 'dpkg'};
+    'unattended-upgrades':
+      paths  => '/var/log/unattended-upgrades/unattended-upgrades.log',
+      tags   => ['unattended'],
+      fields => {'application' => 'apt'};
+    'unattended-upgrades-shutdown':
+      paths  => '/var/log/unattended-upgrades/unattended-upgrades-shutdown.log',
```

Comment:
this could be combined with `unattended-upgrades` above by passing both filenames to the `paths` list